features/jupyter.feature

```diff
@@ -4,6 +4,10 @@ Feature: Jupyter targets in new project
     Given I have prepared a config file
     And I have run a non-interactive kedro new with starter "default"
 
+  Scenario: Execute jupyter-init target
+    When I execute the kedro jupyter command "init"
+    Then I should get a message including "The kernel has been created successfully at"
+
   Scenario: Execute jupyter-notebook target
     When I execute the kedro jupyter command "notebook --no-browser"
```

Comment:
`kedro jupyter` is not an accurate descrption anymore, since `kedro init` isn't using the `jupyter` command. There is another ticket so I put it here as a sidenote.